### PR TITLE
Bump repo-config to v0.6.2

### PR DESCRIPTION
**Pull request by TalweSingh**

### CODE REVIEW

**Comment by TalweSingh** [comment]
@llucax any idea why that one check (ubuntu-20.04, 3.11) is not running?

**Comment by llucax** [comment]
Yes, the `Test with nox` job was split into a new matrix dimension, the *nox session*, so now the name is `CI / Test with nox (ubuntu-20.04, 3.11, **ci_checks_max**) (pull_request) ` and `CI / Test with nox (ubuntu-20.04, 3.11, **pytest_min**) (pull_request)`. The branch protection rules are using the old name, that's why it appears (and waits for it) in the checks. I will update the branch protection rules of the repo.

From https://github.com/frequenz-floss/frequenz-repo-config-python/releases/tag/v0.6.0:

> CI: The nox job now uses a matrix to run the different nox sessions in parallel. If you use branch projection with the nox job you need to update the rules to include each matrix job.

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 1]
```diff
@@ -50,8 +50,6 @@ body:
         - Documentation (part:docs)
         - Unit, integration and performance tests (part:tests)
         - Build script, CI, dependencies, etc. (part:tooling)
-        - The protocol buffer definition files (part:protobuf)
-        - The python bindings (part:python)
```
Comment (llucax) [inline .github/ISSUE_TEMPLATE/bug.yml]: This has to stay

File: .github/keylabeler.yml [hunk 2]
```diff
@@ -16,5 +16,3 @@ labelMappings:
   "part:tests": "part:tests"
   "part:tooling": "part:tooling"
   "part:❓": "part:❓"
-  "part:protobuf": "part:protobuf"
-  "part:python": "part:python"
```
Comment (llucax) [inline .github/keylabeler.yml]: This has to stay too.

File: .github/labeler.yml [hunk 3]
```diff
@@ -13,23 +13,19 @@
...
   - "docs/*.py"
   - CODEOWNERS
   - MANIFEST.in
   - noxfile.py
-
-"part:protobuf":
-  - "proto/**"
-
-"part:python":
-  - "py/**"
```
Comment (llucax) [inline .github/labeler.yml]: And this has to stay too :)

File: .gitignore [hunk 4]
```diff
@@ -1,14 +1,149 @@
-build/*
-dist/*
-py/frequenz_api_dispatch.egg-info/*
-py/frequenz/api/dispatch/**/*_pb2.py
-py/frequenz/api/dispatch/**/*_pb2.pyi
-py/frequenz/api/dispatch/**/*_pb2_grpc.py
-py/frequenz/api/dispatch/**/*_pb2_grpc.pyi
```
Comment (llucax) [inline .gitignore]: These should stay too. We might want to add a more general rule that ignore generated files in the repo-config template though. Feel free to create an issue (or PR!).

File: py/frequenz/api/dispatch/py.typed
Comment (llucax) [inline py/frequenz/api/dispatch/py.typed:1]: Mmm, strange that this was missing... I just checked and it is in `py/frequenz/api/dispatch/v1`, we should remove it from there and only keep it in `py/frequenz/api/dispatch`.

**Review by llucax — DISMISSED** [review]
Thanks for taking on this! Just a few comments.

**Comment by llucax** [comment]
BTW, I will only update the branch protection rules when we are ready to merge, otherwise we'll block merging any other PR.

File: .gitignore [hunk 5]
```diff
@@ -1,14 +1,156 @@
...
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
+
+# C extensions
+*.so
+
+# Auto-generated protobuf files
 py/frequenz_api_dispatch.egg-info/*
```
Comment (llucax) [inline .gitignore:10]: This can go, as it is already ignored by other more general rule, but it doesn't hurt to keep it either.

**Review by llucax — APPROVED** [review]
Just one minor optional comment.

**Comment by TalweSingh** [comment]
PR is queued, @llucax can you update the branch protection rules pls?

**Comment by llucax** [comment]
Done.

**Pull request merged**